Formula/opencoarrays.rb

```diff
@@ -12,14 +12,20 @@ class Opencoarrays < Formula
     sha256 "84cbce876804c2f46cda80bec9b4b310aa0d06300e65473a89207ae6ea8679a3" => :el_capitan
   end
 
+  option "without-failed-image-support", "Disable F2018 failed image support that uses experimental MPI features"
+  option "with-compile-time-tests", "Run the OpenCoarrays test suite at build time."
```

Comment:
We try to limit the number of options, and run tests only for security software.